Packs/Fortimail/Integrations/Fortimail/Fortimail.yml

```diff
@@ -0,0 +1,2214 @@
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.recipient_pattern
+      description: The local part of recipient email address to define recipient (RCPT TO:) email addresses that match this policy.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.recipient_domain
+      description: The domain part of recipient email address to define recipient (RCPT TO:) email addresses that match this policy.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.ldap_profile
+      description: Recipient LDAP profile (use fortimail-ldap-group-list to retrieve all the LDAP profiles). Relevant when recipient_type= LDAP Group.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.recipient_email_address_group
+      description: The recipient email group.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.recipient_pattern_regex
+      description: The recipient email address regular expression pattern.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.antispam
+      description: The name of an outgoing anti spam profile, if any, that this policy will apply.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.content
+      description: The name of the content profile that you want to apply to connections matching the policy. Use fortimail-content-profile-list to retrieve all the content profiles.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.profile_dlp
+      description: The name of the resource profile that you want to apply to connections matching the policy. Use fortimail-resource-profile to retrieve all the resources.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.antivirus
+      description: The name of an antivirus profile, if any, that this policy will apply.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.misc
+      description: The type of the authentication profile that this policy apply.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.auth
+      description: RADIUS authentication profile.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.radius_auth
+      description: LDAP authentication profile.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.ldap_auth
+      description: POP3 authentication profile.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.pop3_auth
+      description: IMAP authentication profile.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.imap_auth
+      description: SMTP authentication profile.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.smtp_auth
+      description: Whether the policy allow email users to log in to their per-recipient spam quarantine by presenting a certificate rather than a user name and password.
```

Comment:
```suggestion
      description: Whether the policy allows email users to log in to their per-recipient spam quarantine by presenting a certificate rather than a user name and password.
```